backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/validator/storage/DiskImagesValidator.java

```diff
@@ -78,17 +78,17 @@ protected Disk getExistingDisk(Guid id) {
     }
 
     /**
-     * Validates that non of the disks exists
+     * Validates that none of the disks exists.
      *
      * @return A {@link ValidationResult} with the validation information.
      */
-    public ValidationResult diskImagesAlreadyExist() {
-
+    public ValidationResult disksNotAlreadyExist() {
```

Comment:
then we better reflect the addition of checking the shareable part here as well, no?

Reply:
Also will need to update the tests after the name change.
Nir would say that it would be a good idea to add shared disks tests as well :)

Reply:
I'm ok with having 2 methods with the same name